charts/backstage/values.yaml

```
@@ -0,0 +1,141 @@
+GreplicaCount: 1
```

Typo, variable name used in `Deployment` is `replicaCount`.

```suggestion
replicaCount: 1
```